pycheribuild/projects/cross/apache.py

```diff
@@ -129,3 +129,36 @@ def configure(self, **kwargs):
                          cwd=self.build_dir / "server")
             self.run_cmd(str(self.host_CC), "gen_test_char.lo", "-o",
                          "gen_test_char", cwd=self.build_dir / "server")
+
+class BuildSSLProc(CrossCompileCMakeProject):
+    target = "sslproc"
+
+    repository = GitRepository("https://github.com/CTSRD-CHERI/sslproc.git")
+
+    native_install_dir = DefaultInstallDir.IN_BUILD_DIRECTORY
+    cross_install_dir = DefaultInstallDir.ROOTFS_LOCALBASE
+
+    @classmethod
+    def setup_config_options(cls, **kwargs):
+        super().setup_config_options(**kwargs)
+        cls.build_tests = cls.add_bool_option("build-tests", help="Build tests",
```

Comment:
This appears to be rather common, I wonder if we should allow something like `has_optional_tests = True` to avoid all this duplication.

Reply:
I've added support for this now so you can do the following instead:
```
    has_optional_tests = True
    default_build_tests = False
    show_optional_tests_in_help = True
```